docs/release_notes/release-3.0.md

```diff
@@ -1,5 +1,32 @@
 # StarRocks version 3.0
 
+## 3.0.6
+
+Release date: September 12, 2023
+
+### New Features
+
+- The aggregate function group_concat() supports distinct and order by. [#28778](https://github.com/StarRocks/starrocks/pull/28778)
+- Data in partitions can be automatically cooled down over time. [#29335](https://github.com/StarRocks/starrocks/pull/29335) [#29393](https://github.com/StarRocks/starrocks/pull/29393)
+
+### Improvements
+
+- Supports implicit conversions for all compound predicates and for all expressions in the WHERE clause. You can enable or disable implicit conversions by using the [session variable](../reference/System_variable.md) `enable_strict_type`. The default value of this session variable is `false`.  [#21870](https://github.com/StarRocks/starrocks/pull/21870)
+- Unifies the logic between FE and BE in converting strings to integers. [#29969](https://github.com/StarRocks/starrocks/pull/29969)
+
+### Bug Fixes
+
+- If `enable_orc_late_materialization` is set to `true`, an unexpected result is returned when a Hive catalog is used to query struct-type data in ORC files. [#27971](https://github.com/StarRocks/starrocks/pull/27971)
+- During data queries through Hive Catalog, if a partitioning column and an OR operator are used in the WHERE clause, the query result is incorrect. [#28876](https://github.com/StarRocks/starrocks/pull/28876)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/1cda8e75-a904-49f5-9ad3-3f654b32aa56)
"used" seems inappropriate. Suggest changing to "specified".